infra/apim_v2/api/selfcare_support_service/v1/open-api.yml.tpl

```diff
@@ -669,6 +669,44 @@ paths:
       security:
         - bearerAuth:
             - global
+  '/api/CountNotifications':
```

Comment:
it might be better to have a path with more context and that follow rest standard....for ex '/onboarding/notifications/count'

Reply:
you're right, i will update it soon but i need to update the target rest endpoint before